Cold Gas Propulsion Analysis and Design

jprks opened the pull request:
### Authors
James Parkus @jprks

### Abstract
This project will extend off the work of the SPEX propulsion team's cold gas thruster, from the 2017 spring semester. The goal of the project is to create a cold gas thruster that functions well and provides experience and data from performing a cold burn. A well functioning cold gas thruster has no hindrances in fluid flow and produces maximum thrust by means of optimal expansion. The experience learned here will help in the ultimate goal of integrating such a system in a CubeSat for the invitation of atmospheric reentry. 

The cold gas thruster will be a closed system which contains all the required regulation and control to sufficiently model a realistic propulsion system for a satellite, this does not refer  to the dimensions of the system or the material requirements. It will model a realistic propulsion system by maximized thrust and remote initiation of the cold burn.

jprks committed:
Merge remote-tracking branch 'refs/remotes/origin/new-organization' into Cold-Gas-Thruster

jprks committed:
updated tex files

jprks committed:
Editted author block

Commented out \thanks section

venku122 commented:
Cleaned up your PR header. There were some weird formatting bugs due to the PDF copy-paste. Also there were several typos, which are fixed in the header, but will need to be addressed in the actual LaTEX document.

Also I swapped the base branch from `master` to `new-organization`. You did everything correct. but we're not quite ready to roll out `new-organization` into master yet.

I'm going through a review now 👍

venku122 commented on Cold-Gas-Thruster/IEEEtran.cls:
```diff
@@ -0,0 +1,4733 @@
+%%
```
This file can be removed. We figured we no longer need it.

venku122 commented on Cold-Gas-Thruster/SPEXformat.cls:
```diff
@@ -0,0 +1,4662 @@
+%% SPEXformat.cls 2017/2/24 version V0.0.1 based on
```
This file can also be removed. In future versions, it won't even be included in the template. :)

jprks commented:
Ok, I will make the changes in the LaTeX document. Yeah the process was a bit confusing for the first time, but as long as this was correct I can probably replicate the process in the future. But the document outlining this process will have to be very clear.

jprks committed:
Fixed the abstract and cls files

Fixed the abstract according to @venku122 suggested edits as well as the
cls files.

venku122 commented:
@jprks Yeah, apologies for making you the guinea pig. Phil and I have already optimized the process a lot since just a few days ago. I hope you and future PDD writers benefit from the review interface.

venku122 committed:
Deleted old figures and unneeded cls files

venku122 commented:
@jprks I went ahead and removed the unneeded files. You'll need to pull the updated branch down before you can commit again.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+
+\begin{abstract}
+  This project will extend off the work of the SPEX propulsion team's cold gas thruster, from the 2017 spring semester.
+  The goal of the project is to create a cold gas thruster that functions well and provides experience and data from performing a cold burn.
```
Reword 'functions well'

runphilrun replied:
I agree. This sentence and the sentence before it can be condensed into one. Needs to be more concise.

jprks replied:
How does this look?
`  This project concerns the design, fabrication, and experimentation of a cold gas thruster. The project's main goal is to provide insight
  and experience into the design of a supersonic nozzle for small propulsion systems. But secondary goal's will include the understanding
  in the design of high-pressure flow systems, including the necessary precautions and procedures to ensure safe usage. The propulsion subsystem
  will provide the thrust and consist of a high pressure tank, a regulator, a solenoid, and the appropriate piping and interfaces. The structure subsystem will
  will hold the propulsion system in place and allow measurement of the thrust and will consist of aluminum sheet metal and t-slotting. This project will be the first
  step in the design of a cold gas thruster for a CubeSat to allow atmospheric re-entry. `

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  A well functioning cold gas thruster has no hindrances in fluid flow and produces maximum thrust by means of optimal expansion.
+  The experience learned here will help in the ultimate goal of integrating such a system in a CubeSat for the invitation of atmospheric reentry.
+
+The cold gas thruster will be a closed system which contains all the required regulation and control to sufficiently model a realistic propulsion system for a satellite,
```
This section can use clarification. A realistic propulsion system for a satellite infers a subsystem that is flight like.

This is a good section to briefly mention the key parts of the project: nozzle assembly, regulators, valves and solenoids that control fluid flow, and the electronic motor controller.

runphilrun replied:
I think the abstract is a little rambly. Abstract are usually no more than one paragraph. You did a good job summing up the paper, so I can understand everything you want to do without needing to reading the whole thing - which is what we want.

Even with adding the clarifications TJ suggested, I do think you can be more concise.

jprks commented:
How do I pull the updated branch down?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  The experience learned here will help in the ultimate goal of integrating such a system in a CubeSat for the invitation of atmospheric reentry.
+
+The cold gas thruster will be a closed system which contains all the required regulation and control to sufficiently model a realistic propulsion system for a satellite,
+this does not refer to the dimensions of the system or the material requirements. It will model a realistic propulsion system by maximized thrust and remote initiation of the cold burn.
```
Can be deleted when the preceding section is clarified.

runphilrun replied:
Agreed

venku122 commented:
@jprks 
There should be a `fetch origin` button on the UI. This is their newest version. The older version should have a similar button.
![fetch_origin](https://user-images.githubusercontent.com/4733447/29234310-1b441e9c-7ec4-11e7-8a76-cbcfdf126b3c.PNG)

jprks committed:
Merge remote-tracking branch 'refs/remotes/origin/new-organization' into Cold-Gas-Thruster

jprks committed:
Merge branch 'Cold-Gas-Thruster' of https://github.com/RIT-Space-Exploration/SPEX-Standard-Proposal into Cold-Gas-Thruster

jprks commented:
Was that it?

venku122 commented:
👍 @jprks

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building, and testing propulsion systems.
+There will be two types of nozzles tested, a minimum length nozzle and an aerospike nozzle. The nozzles will be optimized for ambient conditions.
```
The aerospike nozzle sounds like a great follow-up project, but should be excluded from this PDD specifically. 

Focusing on generating great data with a traditional nozzle will be more than challenging enough for a whole semester. It will also set up the team for success in the Spring to develop new nozzle types like an aerospike.

jprks replied:
Ok, I took that part out. 
`{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building, and testing propulsion systems.
The type of nozzle that will be tested is a minimum length supersonic nozzle. Since there is ambient pressure, the nozzle will be optmized for ambient conditions.`

jprks commented:
Cool. I downloaded the new github desktop too. I like a lot more. So is the next step to just finish the PDD itself?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster capable of sustained cold burns which can be applied to a CubeSat to achieve the reuqired
```
This should be reworded. The primary objective is not to create a CGT for a satellite. It is to develop a CGT that can fired on a ground test stand. This clarification is key.

Always consider the audience for the document. Companies, recruiters, and other aerospace professionals do not have the context of past, present, and future SPEX projects when reading this document.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster capable of sustained cold burns which can be applied to a CubeSat to achieve the reuqired
+velocity to reenter Earth's atmosphere. This is a system property, however using the thrust over time curves reasonable estimations can be made to say whether or not it is possible.
```
This can be deleted.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster capable of sustained cold burns which can be applied to a CubeSat to achieve the reuqired
+velocity to reenter Earth's atmosphere. This is a system property, however using the thrust over time curves reasonable estimations can be made to say whether or not it is possible.
+A target specific impulse is 50s. This can be determined from testing by integrating the thrust over time curve. See the equation 1, below, to see why this is possible.
```
Reword,

`The thruster will be designed for an Isp of 50s. This will be determined from test firings by integrating the thrust of the motor over time according to Equation 1`

Also Isp should be included in the nomenclature section. Also, it is proper to format Isp using an uppercase I and a subscript `sp` to denote its true meaning, Specific Impulse.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\label{fig:lifecycle}
+\end{figure}
+
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data.
```
Clarify the challenges faced in collecting data. What defines `system does not perform well ` and what will be done to mitigate noise.

runphilrun replied:
This is extremely important.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\end{figure}
+
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data.
+However, from previous testing noise is not a problem but error is slightly. The error in the last tests maxed at 0.05 Newtons, which is no terribly significant but still should be accounted.
```
This needs to be reworded.

The reason why noise is not a problem should be expanded upon. How was S/N maximized. What was the S/N ratio itself.

Error should be in a percentage. Giving a force without context of the total force of the thruster and the sensitivity of the sensor is meaningless.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data.
+However, from previous testing noise is not a problem but error is slightly. The error in the last tests maxed at 0.05 Newtons, which is no terribly significant but still should be accounted.
+Vacuum condition availablity is low at RIT, so the team will need to account for ambient conditions. This primarily includes ambient pressure for achieveing maximum thrust, but there are other
+nuisances. The use of this cold gas thruster will give the team experience in the fabrication of propulsion systems and that can be used for designing more complex systems. These systems could
```
Nuisances should be reworded.

The prior line should also be reworded to specify that the nozzles will be designed for sea level operation, not vacuum operation.

runphilrun replied:
Agree. When writing technical documents, try your best to speak in objective terms.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+However, from previous testing noise is not a problem but error is slightly. The error in the last tests maxed at 0.05 Newtons, which is no terribly significant but still should be accounted.
+Vacuum condition availablity is low at RIT, so the team will need to account for ambient conditions. This primarily includes ambient pressure for achieveing maximum thrust, but there are other
+nuisances. The use of this cold gas thruster will give the team experience in the fabrication of propulsion systems and that can be used for designing more complex systems. These systems could
+ include a CubeSat-sized propulsion system.
```
Remove. This should be highlighted in the long term vision section.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+This project would be benefical to SPEX because of the pathway it creates. After its completion, SPEX will have a cold gas thruster that
```
The lead in sentence should clearly present the upcoming arguments. The pathway mentioned in the first sentence is not explained until several lines later.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+% space enthusiasts, and young professionals.
+
+This project would be benefical to SPEX because of the pathway it creates. After its completion, SPEX will have a cold gas thruster that
+be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy, and since
```
What considerations for mobility are included in the design? Is the CGT more mobile than comparable designs?

jprks replied:
I don't think that is something to go into in this section. It does not have to do with the design, it is about how the project benefits SPEX. I don't think this is place for design analysis.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+
+This project would be benefical to SPEX because of the pathway it creates. After its completion, SPEX will have a cold gas thruster that
+be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy, and since
+the system is heavily dependent on chamber pressure for thrust, it is very safe to show to people when the tank is unpressurized. This project
```
This can be removed. Simply stating that the system is safe when not connected to a pressurized gas source will suffice.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+This project would be benefical to SPEX because of the pathway it creates. After its completion, SPEX will have a cold gas thruster that
+be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy, and since
+the system is heavily dependent on chamber pressure for thrust, it is very safe to show to people when the tank is unpressurized. This project
+will also be the first foot in the door for CubeSat propulsion. There is also a continuation of this project ready to be initated upon its completion, the
```
RCS should be moved to the long-term vision section.

venku122 replied:
Bump

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+reaction control system.
+
+There are many levels of
+analysis that can be used to design one, but the best way is experimentation (trial \& error). This is especially true for cold gas propulsion of this
```
Should be reworded. 
 `one` is an unclear pronoun.
`Cold Gas propulsion of this size` should be rephrases as `Cold gas thruster`

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+the nozzle size experiementation must happen. This data will be immensely helpful for designing the RCS nozzles and any other CGT nozzles with similar
+size magnitudes. To do this the team will operate at several different inlet pressures and determine the correct size of these nozzles to create optimal expansion.
+The team will then plot this data and fit a polynomial approximation to it. This polynomial will serve as a initial guess for future nozzles that will be
+expanded upon and optimized during testing.
```
The preceding paragraphs should be in the implementation section, not the benefit to SPEX section.

Benefit to SPEX should be a clear argument of why SPEX should pursue this project. It should relate work items and project outcomes to SPEX goals.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project will include a lot of design and data analysis. Once the design phase
```
This is assumed. Instead, clearly define how much and how long the design phase will be.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % What path do you anticipate the project to take?
+
+This project will include a lot of design and data analysis. Once the design phase
+is complete, testing the system will be the primary concern of the team. The project will start off with a slight redesign of the test stand, optmizing it for new conditions if necessary.
```
What are the modifications to the test stand? How is `slight` defined?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+
+This project will include a lot of design and data analysis. Once the design phase
+is complete, testing the system will be the primary concern of the team. The project will start off with a slight redesign of the test stand, optmizing it for new conditions if necessary.
+Then comes the design of the nozzles. The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
```
👍

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+is complete, testing the system will be the primary concern of the team. The project will start off with a slight redesign of the test stand, optmizing it for new conditions if necessary.
+Then comes the design of the nozzles. The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
+nozzle dimensions to create the polynomial curve approximation that will be an integral part of the rest of the research on the topic. Once that is complete, the
+testing and experimentation phase begins. After the culmination of this phase, the team can try to design, fabricate, and test an aerospike nozzle, only if time permits.
```
Remove and mention in Long Term Vision

venku122 replied:
Remove following references as well.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\subsubsection{Physical Deliverables}
+\label{subsubsec: physical deliverables}
+
+The physical deliverables of this project with the CGT\@. This includes the propulsion system and the test stand. The propulsion system will include a regulator,
```
This sentence is vague. Please remove.

 You should clearly list the physical deliverables and then define them. T

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+The physical deliverables of this project with the CGT\@. This includes the propulsion system and the test stand. The propulsion system will include a regulator,
+a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series. The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. The interfaces between the aluminum parts will also be included. This
+consists of the various t-slot connections and the bolts and washers that connect the aluminum sheets.
+
```
In later sections, you mention electrical and electronic control. Any hardware or software should be specified in this section as well.

jprks replied:
This is the best I can do. I don't know much about how arduinos are used so this is a guess. I can't get David to download and install LaTeX. 

`There will also be an arduino connected to a computer that will use Simulink for data acquisition. The arduino will be a conduit to relay the data from the load cell
to Simulink. There will also be an arduino to control the use of the solenoid. The solenoid will be activated from the Simulink program, which will coincide with 
the beginning of data acquisition. `

venku122 replied:
👍

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+
+The non-physical deliverables of this project will be the project culmination report which will include a very detailed examination of the projects
+results. These results include the expected performance versus the test performance, the nozzle dimension polynomial approximation, the structure performance, and the
+propulsion system performance. The structure performance is basically a section that discusses the reasoning behind the initial design and any problems
```
This section is good, but could be improved by more clearly defining the performance metrics?

What is 'structural performance'? how is it measured?

How will propulsion system performance be measured?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\subsubsection{Design Phase}
+\label{subsubsec: design phase}
+    This will be the first phase and reappearing after the culmination of the experimentation phase.
+    Goals are to design the propulsion system, re-design the test stand, design the minimum length and aerospike nozzle
```
Reword sentence.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\subsubsection{Build \& Preliminary Test Phase}
+\label{subsubsec: sanity test phase}
+    This phase is concerned with the building of the system. The test stand is already built. However a few revisions might be applied, then the team will need
+    to re-build to the new design. The main part of the building stage will be the construction of the propulsion system. This has not been built yet therefore, that
```
This is not clear. Does this mean the existing test stand will receive minor modifications, or will be entirely rebuilt?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\label{subsubsec:experimentation}
+    This phase is primarily concerned with the testing of the CGT\@. This includes gathering all the necessary data and experience to create the
+    polynomial approximation and to aid in the RCS phase and CREST phase of space propulsion research.
+    The testing will consist of 5 tests for each different chamber pressure. These chamber pressures will be; 80, 90, 100, 120, 140, 160, 180, and 200 Psi. There
```
👍

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+    phase for the minimum length nozzle. It is also the phase where the performance analysis takes place. This will inlcude the ultimate calculation of the specific impulse
+    of the thuster and if it attains the values we need to achieve the necessary velocity change for orbital decay to initiate atmospheric reentry.
+
+    \textit{Projected End Date: Week 14 or 15 (not terribly important since the team does not theoretically need to be at RIT to conduct this analysis)}
```
Data analysis and generating the final report must be completed before the end of the semester. If the project requires more time, then it should be allocated more than one semester.

Deliverables and objectives should be clearly marked as mandatory or optional.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+    Once this paper is complete it will be given to the advisors of the SPEX research group for review and then further publications will be discussed. The paper would
+    be great for job interviews to display the work you have done in a formal manner, and one that the interviewer can refer back to at a later date.
+
+    \textit{Projected End Date: End of winter break (the team does not need to be at RIT to write this and they can take advantage of the facilites for testing while they reside there)}
```
See above comment.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+
+    \textit{Projected End Date: End of winter break (the team does not need to be at RIT to write this and they can take advantage of the facilites for testing while they reside there)}
+
+This concludes the milestones for this project.
```
Unnecessary

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+be most qualified for satisifing these requirements are mechanical engineering \& technology and electrical engineering \& technology. But this project is certainly
+open to all majors because most of these skills can be taught in at least a basic manner.
+
+The basic nozzle theory required here is laid out in several documents
```
This section, while helpful is not necessary. Also, most readers will not know or have access to google drive.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+The machining will be the most difficult to learn if there are no MEs or METs that have taken the proper courses. Machining can be extremely dangerous and cannot be done
+without the proper safety training. If no one is trained on the machines one can ask a machinist to make simple parts or submit a job at the Brickmann lab, formula machine shop.
+They have professional engineers there that can make nearly anything one could need. But their timeline slips often, so leave an appropriate amount of time for them to make the part
+so it does not interfere with the projects timeline.
```
For the preceding sections, you should list the required skills, whether members should have those skills before starting the project or whether they can be learned in-situ, and specify where outside help is needed.

Be concise. This section is a resource for reviewers to see exactly what manpower and skill sets are required for a successful project.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Fortunately, most of the materials for the test stand have been purchased. This eliminates much of the system cost. The most costly part of the project
+will be the propulsion system (the regulator, solenoid, nozzle, piping, and interfaces). A preliminary, worst-case, scenario was determined by finding most of the parts
```
Needs to be reworded. 

`A preliminary budget that includes margin for cost overruns...`

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Very limited faculty support will be necessary. The SPEX propulsion team has taken care of most of the parts that will need the assistance from faculty.
+This included the flow system and how it operates. Is it safe and is it necessary? Those questions were answered by Dr. Garrick of the MET department.
```
Do not include hypothetical questions.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+There may be some assistance required if the nozzle performance if far below nominal, but guessing from past work it will not be a problem. There may also
+be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+For the data analysis, if assistance is required for fully understanding it, Dr. Olles would be a good professor to ask. He has a PhD in rocket science
```
Reword for clarification.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Very limited faculty support will be necessary. The SPEX propulsion team has taken care of most of the parts that will need the assistance from faculty.
+This included the flow system and how it operates. Is it safe and is it necessary? Those questions were answered by Dr. Garrick of the MET department.
+There may be some assistance required if the nozzle performance if far below nominal, but guessing from past work it will not be a problem. There may also
```
What guesses and assumptions were made? What are some obstacles and challenges that could lead to poor performance?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+For the data analysis, if assistance is required for fully understanding it, Dr. Olles would be a good professor to ask. He has a PhD in rocket science
+so he could certainly help. Also, a SPEX member, Varun, has experience in flow simulation so he could probably understand the data sufficiently too.
```
Do not specify individual members. PDDs should be timeless, in that they do not depend strongly on the current state of SPEX to be understood.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+
+The long term vision of this project is to be a building block to the next phase of space propulsion research. CREST, a mission that was theorized in the
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
+CGT to assist in the furthering of this project as the complication of the propulsion system has been the main concern. The problem found with this was the required velocity change.
```
Reword for clarity.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
+CGT to assist in the furthering of this project as the complication of the propulsion system has been the main concern. The problem found with this was the required velocity change.
+The CubeSat would need to change its orbital velocity by 122 $ms^{-1}$, opposite its direction of motion. To do this with a cold gas thruster one needs a certain amount
+of propellant varing with the chamber pressure. If you increase the pressure, one can hold more propellant but the optimal value of both was far higher than we anticipated.
```
Too much detail in the follow sections. Be concise when outlining future projects and applications.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+the members to put on their resume. At the end of this project a detailed document will be made which outlines the most important aspects of the project.
+Then the RCS will build off the momentum created by this project. And the understanding of space propulsion inside SPEX will continue to grow and flourish.
+
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
```
👍

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+Then the RCS will build off the momentum created by this project. And the understanding of space propulsion inside SPEX will continue to grow and flourish.
+
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
+frequently in space propulsion and attitude adjustment systems as well as rockets. SpaceX uses them on the falcon 9 for landing orientation adjsutments. You
```
Unnecessary detail

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
+frequently in space propulsion and attitude adjustment systems as well as rockets. SpaceX uses them on the falcon 9 for landing orientation adjsutments. You
+can actually see them burning during landing. It is the burst of white gas coming from the side of rocket. Building an RCS is difficult. But if the team
+has prior experience with cold gas propulsion, their job becomes significantly easier. Because you need optimal nozzles for an RCS, and as mentioned before
```
No need to specify, 'as mentioned before'. Expect readers to remember what you have written.

jprks commented:
Thanks. I will work on them.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+\makenomenclature{}
+
+% set title. choose something as descriptive and precise as possible. Descriptive > sounding cool. remember this!
+\title{An Investigation of a Cold Gas Thruster}
```
I think there's a better title for this out there. "Investigation of a cold gas thruster" makes me think of a case where a thruster malfunctioned and the project is investigating the failure. Perhaps "Design and implementation of a cold gas thruster optimized for sea-level performance" or something along those lines. Should be ideally less than 10 words and inform the core goal or core output of the project as clearly and concisely as possible.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  }
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
```
PDD is not used in the paper. Delete this entry in the Nomenclature

runphilrun replied:
This is not resolved by [`f6e5c2a`](https://github.com/RIT-Space-Exploration/SPEX-Standard-Proposal/pull/16/commits/f6e5c2a174bdfed4fde5bd8cdf2140cee4c652a9). Good practice is to remove this if you do not use it in the body of the text.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,462 @@
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building, and testing propulsion systems.
+There will be two types of nozzles tested, a minimum length nozzle and an aerospike nozzle. The nozzles will be optimized for ambient conditions.
```
`nozzles will be optimized for ambient conditions.`

This is odd for a small thruster designed to be used exclusively for in-space applications. You and I know the reason why you are doing it this way, but it's not obvious. Discuss and justify your reasoning to design the system to operate at ambient pressure.

runphilrun reviewed (commented):
I didn't get through the whole thing, will submit a review later. Formatting looks great, the main focus for changes should be on the clarity and conciseness of the wording.

jprks committed:
Edits

Editted the wording according to the suggestions of @venku122 and @runphilrun

jprks commented:
Made the changes.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -92,14 +92,12 @@
-this does not refer to the dimensions of the system or the material requirements. It will model a realistic propulsion system by maximized thrust and remote initiation of the cold burn.
-
+  This project concerns the design, fabrication, and experimentation of a cold gas thruster. The project's main goal is to provide insight
+  and experience into the design of a supersonic nozzle for small propulsion systems. But secondary goal's will include the understanding
```
Remove 'but'

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -142,10 +146,10 @@
 \label{sec:introduction}
   % The introduction is a place to give background and context before diving into the subject matter.
-  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+  % Establish context forBBB the work you are about to propose and the main ideas of the proposition itself.
```
Typo

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -142,10 +146,10 @@
 
 \IEEEPARstart{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building, and testing propulsion systems.
-There will be two types of nozzles tested, a minimum length nozzle and an aerospike nozzle. The nozzles will be optimized for ambient conditions.
+The type of nozzle that will be tested is a minimum length supersonic nozzle. Since there is ambient pressure, the nozzle will be optmized for ambient conditions.
```
This can be reworded.

'The test nozzle will be a a minimum length converging-diverging nozzle designed for supersonic flow. The nozzle will be optimized for sea-level operating conditions.'

jprks replied:
This is different in the new version. Do you still want me to re-word it?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -154,32 +158,29 @@ \section{Primary Objective}
-velocity to reenter Earth's atmosphere. This is a system property, however using the thrust over time curves reasonable estimations can be made to say whether or not it is possible.
-A target specific impulse is 50s. This can be determined from testing by integrating the thrust over time curve. See the equation 1, below, to see why this is possible.
+The primary objective of this project is to create a cold gas thruster for on-ground experimentation. The goal for the data is to be able to design a nozzle for a small propulsion system, where boundary
+layer effects are dominent, but also to achieve an specific impulse of 50s. Using the thrust over time curves, the specific impulse can be
```
typo ~dominent~ dominant

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -154,32 +158,29 @@ \section{Primary Objective}
+\end{equation}
+
+As seen above, calculating the specific impulse is
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data. The main difficulty in collecting data in the accuracy of
```
This needs to be reworded.

Focus on risk of data collection and methods to mitigate the risk, rather than hoping the risks don't happen. If risks cannot be mitigated, state that those risks will have to be accepted into the design.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -154,32 +158,29 @@ \section{Primary Objective}
+As seen above, calculating the specific impulse is
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data. The main difficulty in collecting data in the accuracy of
+the load cell. The load cell that captured correct data in the Spring of 2017 was accurate within 5\%. This is accurate enough for our forces.
+Vacuum condition availablity is low at RIT, so the team will design the nozzles for ambient conditions.
```
This can be removed.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -194,9 +195,8 @@ \section{Benefit to SPEX}
-be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy, and since
-the system is heavily dependent on chamber pressure for thrust, it is very safe to show to people when the tank is unpressurized. This project
+This project would be benefical to SPEX because of the experience given to SPEX in propulsion system design and experimentation. After its completion, SPEX will have a cold gas thruster that
+be shown to companies, faculty, students, and at any event where SPEX is on display. The CGT's high mobility will make transportation easy and it is very safe to show to people when the tank is unpressurized. This project
```
insert 'can'

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -210,24 +210,19 @@ \section{Benefit to SPEX}
-size magnitudes. To do this the team will operate at several different inlet pressures and determine the correct size of these nozzles to create optimal expansion.
-The team will then plot this data and fit a polynomial approximation to it. This polynomial will serve as a initial guess for future nozzles that will be
-expanded upon and optimized during testing.
+size magnitudes.
```
Should be reworded.

Examples: class, thrust level, characteristics,

jprks replied:
Can you be more specific here. I don't understand what you want.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -210,24 +210,19 @@ \section{Benefit to SPEX}
-This project will include a lot of design and data analysis. Once the design phase
-is complete, testing the system will be the primary concern of the team. The project will start off with a slight redesign of the test stand, optmizing it for new conditions if necessary.
-Then comes the design of the nozzles. The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
+This project will include 4 to 5 weeks of design and 4 to 5 weeks of data analysis. The data analysis will go into winter break because RIT facilites are not needed to analyze data. Once the design phase
```
This is not a wording change.

Would it be possible to complete all analysis during the semester? Final project reports and presentations should be ready by 2nd to last week.

If the project cannot be completed in that time, can the test plan be split into two phases, where an intermediary report is produced by the end of the semester?

jprks replied:
I changed that.

jprks replied:
Should be different in the new revision.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -210,24 +210,19 @@ \section{Benefit to SPEX}
-is complete, testing the system will be the primary concern of the team. The project will start off with a slight redesign of the test stand, optmizing it for new conditions if necessary.
-Then comes the design of the nozzles. The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
+This project will include 4 to 5 weeks of design and 4 to 5 weeks of data analysis. The data analysis will go into winter break because RIT facilites are not needed to analyze data. Once the design phase
+is complete, testing the system will be the primary concern of the team. The project will start off with a redesign discussion of the test stand, optmizing it for new conditions if necessary.
```
~optmizing~ optimizing 

Project scope question:
How much work needs to be done on the existing test stand? How would it affect the timeline of this project?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -210,24 +210,19 @@ \section{Benefit to SPEX}
-There are benefits to aerospikes but there are also disadvantages. The advantages are concerned with the region of optimal expansion. The aerospike nozzle can match the
-ambient pressure over a larger region and therefore can produce more thrust. However, the disadvantages are increased thermal loading, which is not terribly important here but is worht mentioning,
-and high design and fabrication complexity. This is the projected path for the investigation of the cold gas thruster.
+testing and experimentation phase begins. This is the projected path for the investigation of the cold gas thruster.
```
The last sentence is redundant and can be removed.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -237,20 +232,27 @@ \subsection{Deliverables}
-that were found during testing. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
-the mass flow under various ambient conditions and temperatures.
+propulsion system performance. The structure performance is a section that discusses the reasoning behind the initial design and any problems
+that were found during testing. The problems to be avoided will be, unecessary frictional loss due to bad connection points or instability of the test stand due to
```
This should be reworded, and included in a project risk section.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -237,20 +232,27 @@ \subsection{Deliverables}
-propulsion system performance. The structure performance is basically a section that discusses the reasoning behind the initial design and any problems
-that were found during testing. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
-the mass flow under various ambient conditions and temperatures.
+propulsion system performance. The structure performance is a section that discusses the reasoning behind the initial design and any problems
```
~structure~ structural. Structure is not an adjective.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -237,20 +232,27 @@ \subsection{Deliverables}
-the mass flow under various ambient conditions and temperatures.
+propulsion system performance. The structure performance is a section that discusses the reasoning behind the initial design and any problems
+that were found during testing. The problems to be avoided will be, unecessary frictional loss due to bad connection points or instability of the test stand due to
+bad interfacing between structural components. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
```
Structural and propulsion system performance should be renamed for clarity. They are being used to refer to sections within the document, although they commonly refer to the actual characteristics of the system.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -262,15 +264,18 @@ \subsection{Milestones}
 \label{subsubsec: design phase}
-    This will be the first phase and reappearing after the culmination of the experimentation phase.
-    Goals are to design the propulsion system, re-design the test stand, design the minimum length and aerospike nozzle
+    This will be the first phase of the project.
```
Can be removed.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -262,15 +264,18 @@ \subsection{Milestones}
-    This will be the first phase and reappearing after the culmination of the experimentation phase.
-    Goals are to design the propulsion system, re-design the test stand, design the minimum length and aerospike nozzle
+    This will be the first phase of the project.
+    The goals are to design the propulsion system, design the test stand, design the minimum length supersonic nozzle. At the end of this phase
```
Requires a conjunction or rewording.

`The goals of the design phase including designing the propulsion system, test stand, and minimum length nozzle.`

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -262,15 +264,18 @@ \subsection{Milestones}
-    Goals are to design the propulsion system, re-design the test stand, design the minimum length and aerospike nozzle
+    This will be the first phase of the project.
+    The goals are to design the propulsion system, design the test stand, design the minimum length supersonic nozzle. At the end of this phase
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
```
Could be reworded.

`The goal at the end of this phase is to have a full system CAD model, and a series of CAD designs of nozzles ready to be fabricated.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -262,15 +264,18 @@ \subsection{Milestones}
+    This will be the first phase of the project.
+    The goals are to design the propulsion system, design the test stand, design the minimum length supersonic nozzle. At the end of this phase
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
+    will use it as a guide (inlcuding drawings for machinists).
```
This is redundant.

jprks replied:
I do not understand how this is redundant.

venku122 replied:
`This will be the first phase of the project.` can be removed, because it is being described first

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -262,15 +264,18 @@ \subsection{Milestones}
-    to re-build to the new design. The main part of the building stage will be the construction of the propulsion system. This has not been built yet therefore, that
+    This phase is concerned with the building of the system. The test stand is already built. However if minor revisions are necessary, then the team will
+    build the new design (any re-design will concern the dimensions of the tank fixture and will therefore not take long as it is a simplistic system).
+    The main part of the building stage will be the construction of the propulsion system. This has not been built yet, that
```
~This has not been built yet~ can be removed. It is assumed, since it is the project goal.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -342,38 +344,43 @@ \subsection{Prerequisite Skills}
-
-For the data analysis, if assistance is required for fully understanding it, Dr. Olles would be a good professor to ask. He has a PhD in rocket science
-so he could certainly help. Also, a SPEX member, Varun, has experience in flow simulation so he could probably understand the data sufficiently too.
+This included the flow system and how it operates. Dr.Garrick, MET department head, assisted the team in the design of the flow system and pointed out unecessary components, such
```
typo ~unecessary~ unnecessary

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -342,38 +344,43 @@ \subsection{Prerequisite Skills}
-so he could certainly help. Also, a SPEX member, Varun, has experience in flow simulation so he could probably understand the data sufficiently too.
+This included the flow system and how it operates. Dr.Garrick, MET department head, assisted the team in the design of the flow system and pointed out unecessary components, such
+as the flow straightener. He said that the flow was at such a high reynolds number that straightening it in the size constraints is impossible with our budget.
+There may be some assistance required if the nozzle performance if far below nominal, but guessing from past work it will not be a problem. This would concern the nozzle design.
```
~if~ is

This whole section could be split into a new section outlining potential risks and looking ahead at opportunities for risk mitigation.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -342,38 +344,43 @@ \subsection{Prerequisite Skills}
-With a spherical tank the chamber pressure would be near 15000 Psi and with a rectangular tank the pressure would be near 8000 Psi. Because this was the first propulsion system
-we had designed, we were doubtful that NASA would accept these conditions. Therefore, we are making one. This project will also be great for
+CGT to assist in the furthering of this project. The main conern for CREST was the propulsion necessary to de-orbit quickly enough that SPEX can determine its entry location and therefore esitmatt its landing location.
+The problem found with this was the required velocity change.
```
Remove.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -342,38 +344,43 @@ \subsection{Prerequisite Skills}
-of propellant varing with the chamber pressure. If you increase the pressure, one can hold more propellant but the optimal value of both was far higher than we anticipated.
-With a spherical tank the chamber pressure would be near 15000 Psi and with a rectangular tank the pressure would be near 8000 Psi. Because this was the first propulsion system
-we had designed, we were doubtful that NASA would accept these conditions. Therefore, we are making one. This project will also be great for
+CGT to assist in the furthering of this project. The main conern for CREST was the propulsion necessary to de-orbit quickly enough that SPEX can determine its entry location and therefore esitmatt its landing location.
```
~estimatt~ estimate

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -342,38 +344,43 @@ \subsection{Prerequisite Skills}
-has prior experience with cold gas propulsion, their job becomes significantly easier. Because you need optimal nozzles for an RCS, and as mentioned before
-, it is nearly impossible to analytically design them. The team can refer back to their data for this.
+frequently in space propulsion and attitude adjustment systems as well as rockets. Building an RCS is difficult. But if the team
+has prior experience with cold gas propulsion, their job becomes significantly easier. The team can refer back to their data for this.
```
The last sentence should be reworded to clarify 'this'

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -342,38 +344,43 @@ \subsection{Prerequisite Skills}
+
+After the culmination of the cold gas thruster project, the team can try to design, fabricate, and test an aerospike nozzle, only if time permits.
+There are benefits to aerospikes but there are also disadvantages. The advantages are concerned with the region of optimal expansion. The aerospike nozzle can match the
+ambient pressure over a larger region and therefore can produce more thrust. However, the disadvantages are increased thermal loading, which is not terribly important here but is worht mentioning,
```
Reword.

~worht~ worth

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/cold-gas-thruster.png}
+  \caption{This is the Solidworks Assembly of the current design of the SPEX cold gas thruster. The piping interface to the nozzle is omitted for clarity.}
```
* Remove `This is`
* Change `current design` to something more objective and "timeless" I guess. Or just say it was designed in Spring 2017 or something
* What is "the piping interface"?

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+This project would be benefical to SPEX because of the experience given to SPEX in propulsion system design and experimentation. After its completion, SPEX will have a cold gas thruster that
```
We are really hoping to get into what "experience" actually entails. Learning about propulsion theory, practice with prototyping and testing designs, or building test benches... I'd like this to be a bit more detailed if you can. Doesn't have to be exhaustive, but spend a little time thinking about what "experience" actually means to you and the team.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project will include 4 to 5 weeks of design and 4 to 5 weeks of data analysis. The data analysis will go into winter break because RIT facilites are not needed to analyze data. Once the design phase
```
I'm confused about why you mention tasks `going into winter`. You haven't listed a target start date, so all your time ranges are relative. If this work doesn't start until winter then this comment doesn't make sense.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+\label{subsubsec: physical deliverables}
+
+The propulsion system will include a regulator,
+a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series. The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. The interfaces between the aluminum parts will also be included. This
```
Why aluminum? why not steel or plastic?

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series. The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. The interfaces between the aluminum parts will also be included. This
+consists of the various t-slot connections and the bolts and washers that connect the aluminum sheets.
+
+There will also be an arduino connected to a computer that will use Simulink for data acquisition. The arduino will be a conduit to relay the data from the load cell
```
Why not a National Instruments DAQ or something more advanced?

jprks replied:
I would like to, but I do not know for certain that we can get one of those. Are they not expensive? Also, I would like David to handle this section because I know very little about this stuff. I am taking my first class on it this semester.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+bad interfacing between structural components. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
+the mass flow under various ambient conditions and temperatures. The propulsion performance will mostly be measured through the data analysis. This specifically concerns the
+specific impulse. It will also concern the expansion of the flow, as the maximum thrust will occur when the exit pressure equals the ambient pressure and thus the shocks
+in the flow will expand optimally. This will be observed through schlieren lensing, as it has been performed before during nozzle testing.
```
`as it has been performed before`... this needs either a citation or more explanation.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
+    will use it as a guide (inlcuding drawings for machinists).
+
+    \textit{Projected End Date: Week 4 (maybe 5 if schedule slips, but no later)}
```
I'm more a fan of relative dates than absolutes. Say how much time you think the work will take, not when you think it will start/end

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+should not be difficult for a first year or second year EE or EET to set this up. If it proves to be difficult, the team can each out to older students or professors for assistance.
+Dr. Patru would certainly be a great resource for this kind of help.
+
+The basic coding abilites include writing a MATLAB script to analyze and plot data. This is not hard for anyone who has coded before. If it proves to be a roadblock
```
Why MATLAB and not some other language?

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+Dr. Patru would certainly be a great resource for this kind of help.
+
+The basic coding abilites include writing a MATLAB script to analyze and plot data. This is not hard for anyone who has coded before. If it proves to be a roadblock
+one can look up the MATLAB scripts already in the google drive for reference. These scripts layout all required techniques and a bit more, such as an iterative approach.
```
Citation needed.
Maybe be more specific than `the google drive`. Who's drive? I say reword this to avoid saying where they are located.

but yeah, citation needed on which scripts you're referring to

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+one can look up the MATLAB scripts already in the google drive for reference. These scripts layout all required techniques and a bit more, such as an iterative approach.
+
+The machining will be the most difficult to learn if there are no MEs or METs that have taken the proper courses. Machining can be extremely dangerous and cannot be done
+without the proper safety training. If no one is trained on the machines one can ask a machinist to make simple parts or submit a job at the Brickmann lab, formula machine shop.
```
Earl Brinkman Machine Tools and Manufacturing Lab.
What is "the formula shop"? Again, it is best to be specific. The RIT Formula SAE machine shop or Kate Gleason College of Engeering machine shop.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Very limited faculty support will be necessary. The SPEX propulsion team has taken care of most of the parts that will need the assistance from faculty.
```
I'm skeptical that you'll only need limited faculty support. What does it mean that you "took care of" things?

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+\label{sec:vision}
+
+The long term vision of this project is to be a building block to the next phase of space propulsion research. CREST, a mission that was theorized in the
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
```
Citation needed. Change theorized -> proposed

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+Fortunately, most of the materials for the test stand have been purchased. This eliminates much of the system cost. The most costly part of the project
+will be the propulsion system (the regulator, solenoid, nozzle, piping, and interfaces). A preliminary budget that includes margin for cost overruns is \$600.
+The total cost was approximately \$450. That would be the last part of the system needed before testing can occur. Accounting for unforseen cost then rounding upwards,
+the consquentical estimated cost is put at \$600 for the entirty of the project. A bill of materials is displayed in Appendix B.
```
Entirety*

Place the BOM or a table of estimated costs here.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+\section{Cold Gas Thruster Preliminary Bill of Materials}
+
+\begin{figure}[h]
+  \includegraphics[width = \linewidth]{figs/CGT-BoM.png}
```
* I would prefer all tables to be built within LaTeX, but for a table this big that would be a chore.
* Please follow [this table style guide](https://www.inf.ethz.ch/personal/markusp/teaching/guides/guide-tables.pdf) to format your table
* The table is too small. Consider rotating 90 degrees or switching this page to landscape mode it so that it fills the page
* List the vendor and part number rather than a URL.

jprks replied:
I'll type it out. I wasn't too happy with its style anyways. And I'll put the vendor and part number.
But first I am going to complete the bibliography so you can read over the changes(except the BoM table) and see if you are satisfied.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,469 @@
+  \caption{This is the preliminary BoM that the propulsion team came up with last year. Most of the materials have already been acquired. But the propulsion system materials must still be purchased.}
+\label{fig: CGT BoM}
+\end{figure}
+\end{document}
```
Add `\bibliography{your-bib-file-name-here}` before `\end{document}`
This will place the references for citations you might make elsewhere in the PDD

jprks replied:
I made a bib file with the appropriate structure, I think...
But when I added that line it did nothing.

runphilrun reviewed (changes requested):
I read for content this time and left some feedback. I'm not a huge fan of wordage like "This is because" or "This is the" and have opinions on how one should use "the" but I think it's just a matter of preference.

Please add citations for documents that you reference.

jprks committed:
Edits Round 2

Editted the document according the @runphilrun and @venku122 suggested changes

jprks commented:
Made a new round of edits. Phil, I made those tables. Although I am uncertain of how to center the text inside the table, my attempts proved fruitless. I have had no success displaying the bibliography but I have it made. TJ, I made all of your edits too. I may have missed one or two since there are a lot of them so let me know if I skipped over something by accident.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+  \hline
+\end{tabular}
+
+\bibliography{CGT-PDD.bib}
```
The bibliography does not appear because you have not made any citations in your document.
To make a citation, insert the command `\cite{document-name}`, where `document-name` is the name of the entry in your bib file (`nozzles`, for example).

Citations should be made any time you reference work external to this document, or facts that may not be common knowledge. Most research papers do a good job of this, so there are plenty of examples to follow.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+\section{\textbf{Cold Gas Thruster Preliminary Bill of Materials}}
+
+\centering
+\begin{tabular}{ |p{4cm}||p{1.0cm}|p{1.5cm}|p{2.5cm}|p{2.0cm}|p{2.0cm}| }
```
I should make a tables tutorial...

Your text is not centered because of how you have structured the arguments here: 
```tex
\begin{tabular}{these arguments}
```
These arguments tell LaTeX how to format your columns. What you have here tells latex to make a vertical line, then 4cm wide column, vertical line, then 1cm wide column, etc. Generally, we want LaTeX to do the work for us.
```tex
\begin{tabular}{r c l}
```
This says to make the first column right aligned, middle column center aligned, 3rd column left aligned, and to size the cells however best fits the data.

I strongly oppose vertical lines, and even horizontal lines unless separating row headers from the table body. Tables can be structured to present data clearly without unnecessary lines. I linked to a style guide that shows you how to make tables that are easy to read and look good. It includes sample code to use too. Here it is again: https://www.inf.ethz.ch/personal/markusp/teaching/guides/guide-tables.pdf

jprks replied:
The pdf is failing to load. But how the heck can you have a table without lines? I do not understand that.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+try to alleviate these costs.
+
+\centering
+\begin{tabular}{ |p{2cm}|p{2cm}|p{2cm}| }
```
see table comments below

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+
+As seen above, calculating the specific impulse is
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data. The main difficulty in collecting data in the accuracy of
+the load cell. The load cell that captured correct data in the Spring of 2017 was accurate within 5\%. This is accurate enough for our forces.
```
`The load cell that captured correct data in the Spring of 2017 was accurate within 5\%`
Citation needed.

jprks replied:
What kind of citation? We determined this value ourselves.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+As seen above, calculating the specific impulse is
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data. The main difficulty in collecting data in the accuracy of
+the load cell. The load cell that captured correct data in the Spring of 2017 was accurate within 5\%. This is accurate enough for our forces.
+Vacuum condition availablity is low at RIT, so the team will design the nozzles for ambient conditions.
```
Reword "vacuum condition availability"

venku122 commented:
A small suggestions.

You can use the commit message as a way to describe changes you've made in the commit. I usually go with 

```
<statement describing main objective of the commit>

<sentence talking about a change>
...
...
```

Its a good way for you to keep track of what you worked on, without needing to go into the actual code to see what's changed. It also helps people reviewing commits in the future, since they can go through the git log and get a sense for what each commit is doing.

venku122 reviewed (changes requested):
Unfortunately this batch of comments didn't get submitted last time. My apologies.

jprks commented:
I thought I did that.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+in the flow will expand optimally. This will be observed through schlieren lensing, as it has been performed before during nozzle testing. This was performed in the Spring 2017
+semester for flow visualization. The procedure was; turn the lights off, put the nozzle on the air hose, put a white piece of paper on a table, put it parallel to the table, initiate the cold burn, use a phone light
+to project the flow onto the white paper below it. Then distance between the nozzle and the light and the table were toggled until the best focus point was found. That is how the team
+was able to get the picture below.
```
`That is how the team was able to get the picture below.`
Do not reference a figure's position on the page. Instead, reference its figure using `\autoref{fig:my-figure}` where `fig:my-figure` is the label attached to your figure.

I think you can omit that sentence entirely and add something like "shown in `\autoref{fig:nozzle-shocks}`" somewhere in the previous line.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/nozzle-shocks[2].png}
+  \caption{The flow is seen here exiting a small 3D printed supersonic nozzle. The procedure was not unlike schlieren lensing, but the clarity was much lower.}
+  \label{fig: nozzle shocks}
```
Not sure if LaTeX will have issues or not, but common practice is to not use spaces in labels. use hyphens if necessary.

jprks replied:
Ahhh that is why it has been throwing errors at me forever there...

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+
+There is a preliminary plan for the semester that is displayed in further detail in Appendix A. The following list will display the description and timing
```
`in further detail in Appendix A.`
Change this to a cross reference. Hard coding appendix numbers/letters, figure numbers, table numbers, or citations is not correct. If your element label changes in the future, you'd have to change everything manually--that's not what we want.

I think `\autoref` will do what you want. This command comes with the `hyperref` package.

`The following list will display the description and timing`
Omit this sentence.

jprks replied:
It displays Section A instead of Appendix A. Is that ok?

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+be most qualified for satisifing these requirements are mechanical engineering \& technology and electrical engineering \& technology. But this project is certainly
+open to all majors because most of these skills can be taught in at least a basic manner.
+
+The basic nozzle theory required here can be adequately learned by reading the introductory chapters of "Rocket Propulsion Elements" by Sutton. Chapter 1 through 4 will
```
This needs to be a citation and included in your bibliography.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+
+ The CAD software of choice is Solidworks. It is very common for industry and has all the features that the team
+will need throughout the process. The education edition also includes the simulation premium package that can allow the team to run flow simulations if needed. The procedure for
+doing this is complicated but can be learned to a basic degree on youtube.
```
YouTube is a proper noun

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+
+The basic electrical engineering including setting up and arduino or raspberry pi to read out the data from the load cell and turning the solenoid on and off. It
+should not be difficult for a first year or second year EE or EET to set this up. If it proves to be difficult, the team can each out to older students or professors for assistance.
+Dr. Patru would certainly be a great resource for this kind of help.
```
Periods used for abbreviations have different spacing afterward than periods ending sentences. 
Add an interword space after all instances of "Dr.".
Example
```tex
Dr.\ Patru
```

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,510 @@
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Little faculty support will be necessary. The SPEX propulsion team has consulted faculty about structure and flow system concerns and made the suggested changes. Most of the
```
I still disagree.

runphilrun commented on Cold-Gas-Thruster/CGT-PDD.bib:
```diff
@@ -0,0 +1,11 @@
+@article{nozzles,
```
I remember putting several citation needed comments in my last review but where are the citations?

I'm surprised that you only have one document to reference.

runphilrun reviewed (changes requested):
Excellent changes!! Still missing citations and I have issues with table formatting, as always.

jprks committed:
Further edits

Made more grammatical edits, suggested by @venku122

jprks commented:
Ok I made most of the edits.

jprks committed:
Minor Edits

Fixed a centering problem where the text was incorrectly justified after applying the command to a prior table.

runphilrun commented:
When you add citations and fix the table formatting I will approve. (If the table formatting is too annoying, I'll consider approving with the table as-is.)

jprks committed:
Merge branch 'master' into Cold-Gas-Thruster

jprks committed:
Made Suggested Edits

Made edits suggested by @runphilrun
Corrected the table format, added citations, some minor grammtical edits, added nomenclature units

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{CGT}{Cold Gas Thruster}
+\nomenclature{RCS}{Reaction Control System}
+\nomenclature{$I_{sp}$}{Specific Impulse \nomunit{\,\si{s}}}
```
You're misusing the `\si` command from `siunitx`. It will automatically generate your units if you basically type in the whole-word equivalents.

```tex
\nomenclature{$I_{sp}$}{Specific Impulse \nomunit{\,\si{\sec}}}
\nomenclature{$I_t$}{Total Impulse \nomunit{\,\si{\newton\second}}}
\nomenclature{$g_0$}{Standard gravity \nomunit{\,\si{\meter\per\second\squared}}}
\nomenclature{$m_p$}{Propellant Mass \nomunit{\,\si{\kilo\gram}}}
\nomenclature{$t_f$}{Final time \nomunit{\,\si{\second}}}
\nomenclature{$t_i$}{Initial time \nomunit{\,\si{\second}}}
```

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+\label{fig:lifecycle}
+\end{figure}
+
+\begin{equation} \label{specific impulse}
```
You should not have the label `\label{specific impulse}` defined in two separate places. Each label must be unique.

jprks replied:
Yeah I fixed that. I noticed it was complaining about that.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/nozzle-shocks[2].png}
+  \caption{The flow is seen here exiting a small 3D printed supersonic nozzle. The procedure was not unlike schlieren lensing, however the clarity was much lower.}
+  \label{fig: nozzle-shocks}
```
This label does not match its reference on line 269

jprks replied:
Fixed that too.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+
+\usepackage{array}
+
+\usepackage{biblatex}
```
Remove 
```tex
\usepackage{biblatex}
\addbibresource{CGT-PDD.bib}
```

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+\bibliography{CGT-PDD.bib}
+
+\newpage
+\printbibliography
```
Remove
```tex
\newpage
\printbibliography
```

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+ambient pressure over a larger region and therefore can produce more thrust. However, the disadvantages are increased thermal loading and high design and fabrication complexity\cite{SPAD}.
+
+\section*{Acknowledgements}
+The author would like to thank Dr. Garrick and Dr. Olles for their help in understanding the theory of nozzles and the review of the system before testing.
```
There is still improper inter-word spacing after all instances of "Dr."
Do a find and replace 

Find: `Dr. `
Replace with: `Dr.\ `

jprks replied:
I fixed that too.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+
+\section*{Acknowledgements}
+The author would like to thank Dr. Garrick and Dr. Olles for their help in understanding the theory of nozzles and the review of the system before testing.
+
```
Add
```tex
\bibliographystyle{IEEEtran}
\bibliography{CGT-PDD}
```

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
+    will use it as a guide (inlcuding drawings for machinists).
+
+    \textit{Projected Duration: 2-3 weeks}
```
Turns out `-`, `--` and `---` mean different things in LaTeX. `-` is a hyphen, `--` is an *en* dash, and `---` is an *em* dash.

`--` proper in this case when you are referring to a range.
https://tex.stackexchange.com/questions/3819/dashes-vs-vs

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,515 @@
+\appendices{}
+\section{\textbf{Detailed Fall Timeline}}
+\centering
+\begin{tabular}{ c l }
```
This table looks a lot better! I recommend you take a look at the package `booktabs` next time you go about making tables, it helps a lot.

runphilrun reviewed (changes requested):
Make these changes to fix your bibliography. I was forgetting that we also need to tell LaTeX how to *style* the bibliography. For this we use `\bibliographystyle{IEEEtran}`

jprks committed:
Table edits and bibliography

Editted the tables and made a better format. Also, integrated a bibliography and made in-paragraph citations.

jprks committed:
Updated acknowledgements section

Added some thanks to @runphilrun and @venku122 for their help.

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,503 @@
+}
+% page header for pages other than cover page
+\markboth{Project Design Document Standard}%
+{Linden \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
```
Change this to your project title and your last name

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,503 @@
+\end{table}
+
+\bibliographystyle{IEEEtran}
+\bibliography{CGT-PDD}
```
Move this after acknowledgements and before appendix

runphilrun commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,503 @@
+
+\section*{Acknowledgements}
+The author would like to thank Dr.\ Garrick and Dr.\ Olles for their help in understanding the theory of nozzles and the review of the system before testing and
+Phil Lindin and T.J. Tarazevits for creating the PDD process and making it as easy as possible and for their continued help.
```
Linden 🏄

jprks replied:
Oops. Sorry.

runphilrun reviewed (dismissed):
Looks good. A few minor changes to make. Fix those and I approve.

The major ones are
* move the bibliography to before the appendix
* Lines 70-71 change it to be your name and project
* Linden

jprks committed:
Repositioning and minor edits

Repositioned the references and corrected spelling, Lindin -> Linden.

runphilrun commented:
@venku122 this still needs your review

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building,
```
This should be removed. The primary objective covers this. 
```
The introduction is a place to give background and context before diving into the subject matter.
Establish context for the work you are about to propose and the main ideas of the proposition itself.
```

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+
+Data error found from past experimentation was 5\% of the thrust. This error is due to the quality of the load cell. It has an inherent inaccuracy. This
+amount of error is acceptable. The performance analysis will not be detrimentally affected. The risk mitigation includes zeroing the load cell just before the test.
+This will erase the effects caused by initial weight placed on the load cell. This initial weight comes from the aluminum sliding fixture which holds the nozzle
```
The second sentence is redundant and can be removed. Most of this section could be reworded to indicate the percentage of error expected and an explanation of why that error will not interfere with test results.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+This project can be benefical to SPEX because of the experience given to SPEX in propulsion system design and experimentation. The experience includes learning how to contruct flow systems, design nozzles
```
Reword for clarity

>This project can be beneficial to SPEX because

is redundant since the section header is `Benefit to SPEX`.  Start with a strong statement and provide supporting evidence in following sentences.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+to the fact that their is no chemcial reaction or heat applied to the system, maxmizing the thrust is imperitive.
+
+Last, but certainly not least, is the nozzle determination experience. Because of the flow conditions and dimensions, an analytical solution will not be sufficient. To accurately determine
+the nozzle size experiementation must happen. This data will be immensely helpful for designing the RCS nozzles and any other CGT nozzles with similar
```
~experiementation~ experimentation

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+
+Last, but certainly not least, is the nozzle determination experience. Because of the flow conditions and dimensions, an analytical solution will not be sufficient. To accurately determine
+the nozzle size experiementation must happen. This data will be immensely helpful for designing the RCS nozzles and any other CGT nozzles with similar
+size magnitudes.
```
Could be rephrased.

>helpful for designing the RCS nozzles and any other CGT nozzles of a similar class.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project will include 4 to 5 weeks of design, 3 to 5 weeks of fabrication, and 4 to 5 weeks of data analysis. The project will start off with a redesign discussion of the test stand, optmizing it for new conditions if necessary.
```
~optmizing~ optimizing 

Do you have any idea what these optimizations might entail?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+This project will include 4 to 5 weeks of design, 3 to 5 weeks of fabrication, and 4 to 5 weeks of data analysis. The project will start off with a redesign discussion of the test stand, optmizing it for new conditions if necessary.
+The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
+nozzle dimensions to create the polynomial curve approximation that will be an integral part of the rest of the research on the topic. Once that is complete, the
+testing and experimentation phase begins. At the end of the data analysis the final report will be the main focus. The aim is to complete it by the beginning of winter break
```
This can be reworded

>At the end of the data analysis process, the team will generate a final report including test procedures, design choices, and test data.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+
+The propulsion system will include a regulator, a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series.
+The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. Aluminum is a great material for its balance of strength, machinability, and
+weight. Steel is very difficult and hardous to machine and is very heavy, although its strength is greater than aluminum. Plastics are tedious to machine as they tend to melt to the cutting
```
The material description can be deleted.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+The propulsion system will include a regulator, a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series.
+The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. Aluminum is a great material for its balance of strength, machinability, and
+weight. Steel is very difficult and hardous to machine and is very heavy, although its strength is greater than aluminum. Plastics are tedious to machine as they tend to melt to the cutting
+blade and that casues the machining to stop frequently to clean off the blade. The propulsion has a very large pressure drop and this will cause a large temperature drop. A plastic structure will
```
>The propulsion has a very large pressure drop and this will cause a large temperature drop

This can be removed

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+to Simulink. There will also be an arduino to control the use of the solenoid. The solenoid will be activated from the Simulink program, which will coincide with
+the beginning of data acquisition.
+
+The system will also be displayed at Imagine RIT 2018. This will include a full poster and data representation.
```
This should be moved to non-physical deliverables

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+propulsion system performance. The structural performance is a section that discusses the reasoning behind the initial design and any problems
+that were found during testing. The problems to be avoided will be, unecessary frictional loss due to bad connection points or instability of the test stand due to
+bad interfacing between structural components. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
+the mass flow under various ambient conditions and temperatures. The propulsion performance will mostly be measured through the data analysis. This effects the
```
> The propulsion performance will mostly be measured through the data analysis.

This is unclear and needs to be reworded

>The performance of the complete propulsion system will be determined through analysis of test result data.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+the mass flow under various ambient conditions and temperatures. The propulsion performance will mostly be measured through the data analysis. This effects the
+specific impulse. It will also concern the expansion of the flow, as the maximum thrust will occur when the exit pressure equals the ambient pressure and thus the shocks
+in the flow will expand optimally. This will be observed through schlieren lensing, as it has been performed before during nozzle testing. This was performed in the Spring 2017
+semester for flow visualization. The procedure was; turn the lights off, put the nozzle on the air hose, put a white piece of paper on a table, put it parallel to the table, initiate the cold burn, use a phone light
```
This can be removed

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+\subsubsection{Design Phase}
+\label{subsubsec: design phase}
+    The goals are to design the propulsion system, the test stand, the nozzle. At the end of this phase
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
```
Redundant. Remove
> This CAD model is essential for the next phase, as the fabrication will use it as a guide (~inlcuding~including drawings for machinists).

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+\subsubsection{Data Analysis}
+\label{subsubsec: data analysis}
+    This phase is concerned with the analysis of the data collected from the CGT testing. The polynomial approximation will be completed during this
+    phase for the minimum length nozzle. It is also the phase where the performance analysis takes place. This will inlcude the ultimate calculation of the specific impulse
```
~inlcude~

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+\label{subsubsec: data analysis}
+    This phase is concerned with the analysis of the data collected from the CGT testing. The polynomial approximation will be completed during this
+    phase for the minimum length nozzle. It is also the phase where the performance analysis takes place. This will inlcude the ultimate calculation of the specific impulse
+    of the thuster and if it attains the values we need to achieve the necessary velocity change for orbital decay to initiate atmospheric reentry.
```
>we need to achieve the necessary velocity change for orbital decay to initiate atmospheric reentry.

Please remove. This project will not concern a system with a calculable delta-v. This should be changed to the target Isp of 50s

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+
+\subsubsection{Final Report}
+\label{subsubsec: final report}
+    At the end of this project a final report will be made to report the outcome. It will outline the design process, fabrication process, and testing process.
```
>At the end of this project a final report will be made to report the outcome.

This is redundant. It can be reworded
>The project will conclude with the production of a final report.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+
+ The CAD software of choice is Solidworks. It is very common for industry and has all the features that the team
+will need throughout the process. The education edition also includes the simulation premium package that can allow the team to run flow simulations if needed. The procedure for
+doing this is complicated but can be learned to a basic degree on YouTube.
```
Please remove

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+will need throughout the process. The education edition also includes the simulation premium package that can allow the team to run flow simulations if needed. The procedure for
+doing this is complicated but can be learned to a basic degree on YouTube.
+
+The basic electrical engineering including setting up and arduino or raspberry pi to read out the data from the load cell and turning the solenoid on and off. It
```
Not grammatically correct

>Basic circuits knowledge will be required to setup the load cell and solenoid to the microcontroller.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+Dr.\ Patru would certainly be a great resource for this kind of help.
+
+The basic coding abilites include writing a MATLAB script to analyze and plot data. MATLAB is easy to learn and is very powerful. It is a very common choice for data analysis for mechancial engineering
+projects where the core skill set is not computer science. This is not hard for anyone who has coded before.
```
>This is not hard for anyone who has coded before.

Please remove

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+will be the propulsion system (the regulator, solenoid, nozzle, piping, and interfaces). A preliminary budget that includes margin for cost overruns is \$600.
+The total cost was approximately \$450. That would be the last part of the system needed before testing can occur. Accounting for unforseen cost then rounding upwards,
+the consquentical estimated cost is put at \$600 for the entirty of the project. A bill of materials is displayed in Appendix B. The table, displayed below, outlines
+the most expensive parts that the team will need to purchase for the propulsion system, these parts are worst-case scenario in cost and measures will be taken to
```
This section can be shortened, in that estimated Bill of Materials cost is $450 but $600 is requested to cover unexpected costs.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Facculty support will be necessary. The SPEX propulsion team has consulted faculty about structure and flow system concerns and made the suggested changes. However, the propulsion system
```
~Facculty~ Faculty

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Facculty support will be necessary. The SPEX propulsion team has consulted faculty about structure and flow system concerns and made the suggested changes. However, the propulsion system
+has not been tested. It is likely that faculty assistance will be necessary to get the propulsion to a functional stage. This would most likely inlcude the design of an optimally expanding
```
~the propulsion~ the propulsion system.

propulsion is not a noun in this case.

venku122 replied:
~inlcude~ include

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+Facculty support will be necessary. The SPEX propulsion team has consulted faculty about structure and flow system concerns and made the suggested changes. However, the propulsion system
+has not been tested. It is likely that faculty assistance will be necessary to get the propulsion to a functional stage. This would most likely inlcude the design of an optimally expanding
+nozzle. Faculty support was needed during the endevour to fabricate the structure and design the flow system.
+This included the flow system and how it operates. Dr.\ Garrick, MET department head, assisted the team in the design of the flow system and pointed out unnecessary components, such
```
This detail is unnecessary. Prior work, detailed at the per person level does not relate to the new project definition.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+nozzle. Faculty support was needed during the endevour to fabricate the structure and design the flow system.
+This included the flow system and how it operates. Dr.\ Garrick, MET department head, assisted the team in the design of the flow system and pointed out unnecessary components, such
+as the flow straightener. He said that the flow was at such a high reynolds number that straightening it in the size constraints is impossible with our budget.
+There may be some assistance required if the nozzle performance is far below nominal, but guessing from past work it will not be a problem. This would concern the nozzle design.
```
What assumptions are you making in this guess?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+on a computer would include high level numerical smulation which beyond the team's skill set. If there are recurring problems here, Dr.\ Olles can be consulted as he has experience with this.
+There may also be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+If assistance is needed for the data analysis, Dr.\ Olles would be a good professor to ask. He has a PhD in rocket science
```
Is this a correct certification?

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
+CGT to assist in the furthering of this project. The main conern for CREST was the propulsion necessary to de-orbit quickly enough that SPEX can determine its entry location and therefore estimate its landing location.
+
+This project will also be great for
```
This can be removed. It is covered in Benefit to SPEX

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+CGT to assist in the furthering of this project. The main conern for CREST was the propulsion necessary to de-orbit quickly enough that SPEX can determine its entry location and therefore estimate its landing location.
+
+This project will also be great for
+the members to put on their resume. At the end of this project a detailed document will be made which outlines the most important aspects of the project.
```
This is redundant. Please remove. It is covered in Deliverables.

venku122 commented on Cold-Gas-Thruster/Cold Gas Thruster PDD.tex:
```diff
@@ -0,0 +1,501 @@
+the members to put on their resume. At the end of this project a detailed document will be made which outlines the most important aspects of the project.
+Then the RCS will build off the momentum created by this project. And the understanding of space propulsion inside SPEX will continue to grow and flourish.
+
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
```
This whole section needs to be significantly shortened. RCS deserves its own PDD and should only be directly mentioned in this document, not thoroughly explained.

jprks committed:
Changed title, and edits

Changed title of paper. Made suggested edits by @venku122 .

runphilrun reviewed (approved):
🏄
